macros/all_passthrough_column_check.sql

```diff
@@ -0,0 +1,10 @@
+{% macro all_passthrough_column_check(relation, get_columns) %}
+
+{% set available_passthrough_columns = fivetran_utils.remove_prefix_from_columns(
+                columns=adapter.get_columns_in_relation(ref(relation)), 
+                prefix='property_', exclude=get_macro_columns(get_columns))
+            %}
+
+{{ return(available_passthrough_columns|int) }}
```

Comment:
could you talk through what the `int` conversion means?

Reply:
Sure thing! The int is needed because the operation by default converts the length to a string. So this is needed to make it an int and use in the conditional to check if it is greater than 0.

Reply:
Without that conversion you get the following error.
![image](https://user-images.githubusercontent.com/74217849/228606631-118344ae-7d80-4c27-98d4-e5d12ad03e06.png)

Reply:
great! marking as resolved